documentation/README_Master_Board_Communication.md

```diff
@@ -0,0 +1,79 @@
+# Master Board Communication
+
+The master board can be connected to a real time control computer via a direct Ethernet link, or via a special wifi interface.
+
+Ethernet
+--------
+Ethernet connection with the robot is happening at the lowest level possible (MAC level without TCP/IP). You cannot use any switch or hub, but only a direct cable between the robot and a dedicated network interface. LAAS has tested the model (TODO ref) with a RT Preempt patched OS.
+
+Prefer Ethernet cable without grounding to avoid to inject grid noise in the robot. (Plastic RJ45 connector)
+
+![rj45](../images/rj45_no_gnd.png)
+
+### Lighten the cable!
+
+On the 4 pairs of wires, only 2 are used by the ethernet 100Mbps and cables are normally shielded.
+
+At LAAS we made light weight cables using only 2 twisted pairs of wire. We attached the wires to the power cables and did not have any issues of packet loss.
```

Comment:
We might want to rephrase some of the sentences, and remove reference to LAAS when it is not needed.
something like "It is possible to make the ethernet cables lighter by..."